apps/betterangels-backend/accounts/types.py

```diff
@@ -130,12 +145,14 @@ class UserBaseType:
 
 @strawberry_django.type(User)
 class UserType(UserBaseType):
+    # TODO: has_accepted_tos, has_accepted_privacy_policy, is_outreach_authorized shouldn't be optional.
+    # Temporary fix while we figure out type generation
     id: ID
-    username: auto
+    has_accepted_tos: Optional[bool]
```

Comment:
`has_accepted_tos`, `has_accepted_privacy_policy`, `is_outreach_authorized` shouldn't be optional. Temporary fix while we figure out type generation. Without `Optional`, typescript requires user queries to include these fields on the request (including queries wherein `user` is a field, e.g., client profile, note)